client.go

```diff
@@ -85,20 +91,24 @@ func (client *Client) Validate() (bool, error) {
 	}
 
 	var resp *http.Response
-	resp, err = client.HttpClient.Do(req)
-	if err != nil {
-		return false, err
-	}
 
-	defer resp.Body.Close()
```

Comment:
Why not defer closing `resp.Body`?

Reply:
I was getting errors in unmarshaling JSON during testing, I think it was because I had it in the wrong scope. I've added it back.